packages/terra-navigation-side-menu/src/MenuItem.module.scss

```diff
@@ -112,7 +112,7 @@
 
       &:hover,
       &.is-hovered {
-        background-color: var(--terra-navigation-side-menu-item-selected-hover-background-color, #f4f4f4);
+        background-color: var(--terra-navigation-side-menu-item-selected-hover-background-color, darken(#ebf6fd, 2%));
```

Comment:
Does the darken() function has same effects on all the browsers?

Reply:
This color codes are taken from the other terra components(terra-list). it has been validated before, so it should be working.